custom_components/myenergi/sensor.py

```diff
@@ -306,14 +306,6 @@ async def async_setup_entry(hass, entry, async_add_devices):
 
         # Sensors common to Zapi and Eddi
         if device.kind in [ZAPPI, EDDI]:
-            sensors.append(
-                MyenergiSensor(
-                    coordinator,
-                    device,
-                    entry,
-                    create_meta("Status", "status", None, None, None, "mdi:ev-station"),
-                )
-            )
```

Comment:
Looks like this was accidentally put in the wrong place in 362985be610659b6e03712474b226ae3c10ca82b